oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/util/ThrottlingDocumentStoreWrapper.java

```diff
@@ -190,6 +191,20 @@ public <T extends Document> T findAndUpdate(final Collection<T> collection, fina
         return oldDoc;
     }
 
+    @Override
+    @NotNull
+    public <T extends Document> List<T> findAndUpdate(@NotNull Collection<T> collection, @NotNull List<UpdateOp> updateOps) {
+        final long throttlingTime = performThrottling(collection);
+        final List<T> results = new ArrayList<>();
+        try {
+            results.addAll(store.findAndUpdate(collection, updateOps));
+        } finally {
+            throttlingStatsCollector.doneFindAndModify(MILLISECONDS.toNanos(throttlingTime), collection,
+                    updateOps.stream().map(UpdateOp::getId).collect(toList()), true, 0);
+        }
+        return results;
```

Comment:
Adding documents to `results` is unnecessary.
```suggestion
        final long throttlingTime = performThrottling(collection);
        try {
            return store.findAndUpdate(collection, updateOps);
        } finally {
            throttlingStatsCollector.doneFindAndModify(MILLISECONDS.toNanos(throttlingTime), collection,
                    updateOps.stream().map(UpdateOp::getId).collect(toList()), false, true, 0);
        }
```